docs/product-business-resource-planner-guide/src/main/asciidoc/optimizer-download-proc.adoc

```diff
@@ -1,23 +1,28 @@
 [id='optimizer-download-proc']
 = Download {PLANNER}
 
-{PLANNER} is production ready.
-The API is stable but backward incompatible changes can occur.
+{PLANNER} is production ready, and can be used to optimize business solutions in {CENTRAL}, in your preferred IDE, or on OpenShift.
 
 .Prerequisite
 The {PRODUCT} has been downloaded and installed from the https://access.redhat.com[Red Hat Customer Portal]. See Installing Decision Manager for more information about installing {PRODUCT}.
 //@doc-link:Add link to Installation assembly above. 
 
 .Procedure
-. Download the {PLANNER} file from the https://access.redhat.com[Red Hat Customer Portal].
-. Unzip the {PLANNER} files.
+. Download and install {PRODUCT} from the https://access.redhat.com[Red Hat Customer Portal]. See the _Getting started with decision services_ for more information about installing {PRODUCT}.
+. Download the {PLANNER} from the https://access.redhat.com[Red Hat Customer Portal].
+. Extract the {PLANNER} files to access the complete set of {PLANNER} engine JAR files. 
+. xref:optimizer-running-the-examples-proc[Run the examples].
+
+
+{PLANNER} is production ready.
+The API is stable, however, backward incompatible changes can occur.
```

Comment:
I'm not sure if we need to mention this here. All our public APIs are stable unless stated otherwise. Backward incompatible changes can only occur in major releases. What's your opinion on this sentence, @ge0ffrey?

Reply:
This came up in a previous PR. 

I'm going to change it completely to just point users to the migration story where we are including a link to the upgrade recipe archive.

"See {URL_MIGRATING_BRMS_TO_DM}[_{MIGRATING_BRMS_TO_DM}_] for information about migrating from previous versions of {PLANNER}."